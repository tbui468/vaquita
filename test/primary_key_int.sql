create database sol;
open sol;

create table planets (id int key, name string, mass int, atmosphere bool);
insert into planets (id, name, mass, atmosphere) values 
    (3, "Mars", 10, true), 
    (-2, "Venus", 10, false),
    (1, "Neptune", 20, false);
select * from planets;
drop table planets;

close sol;
drop database sol;
exit;
