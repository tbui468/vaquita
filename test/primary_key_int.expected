created database sol
opened database sol
created table planets
inserted 3 record(s) into planets
-2, Venus, 10, false, 
1, Neptune, 20, false, 
3, Mars, 10, true, 
dropped table planets
closed database sol
dropped database sol
